packages/vuetify/src/services/breakpoint/index.ts

```diff
@@ -2,9 +2,9 @@
 import { Service } from '../service'
 
 // Types
-import { VuetifyBreakpointOptions } from 'vuetify/types/services/breakpoint'
+import { BreakpointOptions, Breakpoint as VuetifyBreakpoint } from 'vuetify/types/services/breakpoint'
 
-export class Breakpoint extends Service {
+export class Breakpoint extends Service implements VuetifyBreakpoint {
```

Comment:
Any reason to not name this IBreakpoint similar to https://github.com/vuetifyjs/vuetify/pull/7878/files#diff-b498af033ab2b85cd6fba42f81da2562R7